### PR TITLE
Adding a basic healthcheck

### PR DESCRIPTION
**Pull request by malinthaprasan**
Related: https://github.com/wso2-enterprise/choreo/issues/18997

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.internal.service/src/gen/java/org/wso2/carbon/apimgt/internal/service/HealthzApi.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
+package org.wso2.carbon.apimgt.internal.service;
```
Comment (ruwiniwj) [inline components/apimgt/org.wso2.carbon.apimgt.internal.service/src/gen/java/org/wso2/carbon/apimgt/internal/service/HealthzApi.java:1]: don't we need license header?

File: components/apimgt/org.wso2.carbon.apimgt.internal.service/src/main/java/org/wso2/carbon/apimgt/internal/service/impl/HealthzApiServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
```
Comment (ruwiniwj) [inline components/apimgt/org.wso2.carbon.apimgt.internal.service/src/main/java/org/wso2/carbon/apimgt/internal/service/impl/HealthzApiServiceImpl.java:4]: LLC

Comment (malinthaprasan) [inline components/apimgt/org.wso2.carbon.apimgt.internal.service/src/main/java/org/wso2/carbon/apimgt/internal/service/impl/HealthzApiServiceImpl.java:4]:
```suggestion
 * WSO2 LLC. licenses this file to you under the Apache License,
```

Comment (malinthaprasan) [reply components/apimgt/org.wso2.carbon.apimgt.internal.service/src/main/java/org/wso2/carbon/apimgt/internal/service/impl/HealthzApiServiceImpl.java]: Fixed & thanks

**Review by jaadds — APPROVED** [review]

**Pull request merged**